probe/probe.go

```diff
@@ -356,6 +356,7 @@ type KubeArmorProbeData struct {
 	ContainerSecurity       bool
 	ContainerDefaultPosture tp.DefaultPosture
 	HostDefaultPosture      tp.DefaultPosture
+	HostVisibility          string
```

Comment:
Thanks a lot for the PR. I had a major query tho. Are we receiving this data? 

I checked our code in KubeArmor upstream at https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/core/karmorprobedata.go and I don't see where do we actually write to file about HostVisibility.